exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -49,16 +50,16 @@ def int_or_none(value: str) -> Optional[int]:
 
 
 class SpawnTestDockerDatabase(DockerBaseTask, DockerDBTestEnvironmentParameter):
-    environment_name = luigi.Parameter()  # type: str
-    db_container_name = luigi.Parameter()  # type: str
-    attempt = luigi.IntParameter(1)  # type: int
-    network_info = JsonPickleParameter(DockerNetworkInfo, significant=False)  # type: DockerNetworkInfo
-    ip_address_index_in_subnet = luigi.IntParameter(significant=False)  # type: int
-    docker_runtime = luigi.OptionalParameter(None, significant=False)  # type: str
-    certificate_volume_name = luigi.OptionalParameter(None, significant=False)
-    additional_db_parameter = luigi.ListParameter()
-    ssh_user = luigi.Parameter("root")
-    ssh_key_file = luigi.OptionalParameter(None, significant=False)
+    environment_name : str = luigi.Parameter()  # type: ignore
+    db_container_name : str = luigi.Parameter()  # type: ignore
+    attempt : int = luigi.IntParameter(1)  # type: ignore
+    network_info : DockerNetworkInfo = JsonPickleParameter(DockerNetworkInfo, significant=False)  # type: ignore
+    ip_address_index_in_subnet : int = luigi.IntParameter(significant=False)  # type: ignore
+    docker_runtime : Optional[str] = luigi.OptionalParameter(None, significant=False)  # type: ignore
+    certificate_volume_name : Optional[str] = luigi.OptionalParameter(None, significant=False) # type: ignore
+    additional_db_parameter : List[str] = luigi.ListParameter() # type: ignore
+    ssh_user : str = luigi.Parameter("root") # type: ignore
+    ssh_key_file : str | Path | None = luigi.OptionalParameter(None, significant=False) # type: ignore
```

Comment:
`ssh_key_file` is re-used and overwritten several times: sometimes it gets a string assigned, sometimes a Path.

Reply:
oh god, then it is this way

Reply:
@tomuben please open a ticket that we fix ssh key stuff

Reply:
Created #431